acceptance-tests/src/test/scala/ch/renku/acceptancetests/ReplShell.scala

```diff
@@ -1,8 +1,5 @@
-import org.openqa.selenium.{By, WebDriver}
-import org.openqa.selenium.chrome.ChromeOptions
-import org.openqa.selenium.remote.RemoteWebDriver
```

Comment:
This is a weird class because it is here to be used in the Scala REPL. It's true that the imports are not used in the code, but they are here so the names are available in an interactive REPL session, where they are useful. 

I think they should stay, but if you have a better way of expression that intention, then that would of course be great.

Reply:
I did some import optimization and it looks like they got removed :)

Reply:
@travissarles I think we forgot to bring the imports back.

Reply:
Why do we need them? it seems like it still compiles